fiaas_skipper/config.py

```diff
@@ -45,8 +45,9 @@ def _parse_args(self, args):
                             default="http://fiaas-release.delivery-pro.schibsted.io")
         parser.add_argument("--release-channel-metadata",
                             help="Provide hardcoded release channel metadata (Used for debugging purposes).")
-        parser.add_argument("--spec-file", help="Spec file describing how applications should be deployed.",
-                            default=DEFAULT_SPEC_FILE)
+        parser.add_argument("--spec-file-extension",
```

Comment:
Reading this configuration flag makes me think that it has something to do with file extensions. Could something like for example `--spec-file-overrides` or `--spec-overrides-file` be more descriptive?

Reply:
When you mention it, I'm a bit unsure if I like `spec-file` too.. I realize the objects we create when parsing that file in fiaas-deploy-daemon are called `something_spec`, but I've always thought of the file as "fiaas config"... 
Anyway, I like "override" better than "extension", and we can debate "spec" further elsewhere (it's a change reaching far outside this PR). I'll fix.